app/component/itinerary/Itinerary.js

```diff
@@ -114,11 +115,23 @@ export function RouteLeg(
       />
     );
   } else {
+    // Because className uses leg.mode instead of just mode, the externalFeed boolean is needed separately
+    // instead of using it through getRouteMode. It is unclear whether mode could be used for both mode
+    // and className.
+    const externalFeed = isExternalFeed(
```

Comment:
Mode should be used - bus-express etc have their own style classes., see sass/base/_helper-classes.scss.
If something breaks then it is a bug and should  be fixed.